### DISCUSSION TITLE
Finding out the formr version number

### DISCUSSION THREAD

**Discussion by Erefin** [rubenarslan/formr.org#536] [Q&A] [answered]
Dear all,

I've set up and succesfully conducted a study in formr, from March 2023 to January 2024. Where can I find information regarding which version number of formr I've used for my study (so I can cite it in my paper)?

Cheers
Felix

**Comment by rubenarslan** [comment, marked as answer]
In the bottom right corner of the footer of the formr admin area. 

> Copyright © 2024 formr - v0.20.6

If the study was conducted long ago and you didn't check then, you can check this page for release dates. Usually, releases are rolled out to formr.org shortly.

https://github.com/rubenarslan/formr.org/releases

**Reply by Erefin** [reply]
Thank you, Ruben :-)
